bundler/spec/support/helpers.rb

```diff
@@ -23,12 +23,6 @@ def reset_paths!
       Gem.clear_paths
     end
 
-    def self.bang(method)
-      define_method("#{method}!") do |*args, &blk|
-        send(method, *args, &blk)
-      end
-    end
-
```

Comment:
@simi With a quite a few changes to only use the "non bang methods", we can now kill this (now dummy) helper.

Reply:
:+1: